common/display/displayplanemanager.cpp

```diff
@@ -212,10 +212,10 @@ bool DisplayPlaneManager::ValidateLayers(
           if (fall_back) {
             if (!validate_final_layers)
               validate_final_layers = !(last_plane.GetOffScreenTarget());
-
-            ResetPlaneTarget(last_plane, commit_planes.back());
           }
 
+          ResetPlaneTarget(last_plane, commit_planes.back());
```

Comment:
hmm this forces offscreen composition and we should do it only when layer cannot be scanned out (i.e. if fallback is true)